src/interfaces/alert-i.ts

```diff
@@ -0,0 +1,9 @@
+import type { AlertTypesE, AlertSeverityE } from '#imports'
+
+export interface AlertI {
+  severity: AlertSeverityE | undefined
+  alertType: AlertTypesE | undefined
+  text: string | undefined
+  description: string | undefined
+  date: any | undefined
```

Comment:
Note for future: shorthand would be like *severity?: AlertSeverityE* , but I think you reference this interface with Partial anyways so I don't think you need to specify the undefined possibility here